WORKSPACE

```diff
@@ -311,7 +311,7 @@ maven_install(
 )
 
 maven_install(
-    name = "jetify_test",
+    name = "jetify_all_test",
```

Comment:
`jetify_without_include_list_test`? Anything is good really.

Reply:
I think _all captures the intention really good here